pkg/auth/dependency/newinteraction/intents/authenticate.go

```diff
@@ -147,16 +147,11 @@ func (i *IntentAuthenticate) DeriveEdgesForNode(ctx *newinteraction.Context, gra
 		}, nil
 
 	case *nodes.NodeEnsureVerificationEnd:
-		if node.NewAuthenticator != nil {
-			return []newinteraction.Edge{
-				&nodes.EdgeDoVerifyIdentity{
-					Identity:         node.Identity,
-					NewAuthenticator: node.NewAuthenticator,
-				},
-			}, nil
-		}
 		return []newinteraction.Edge{
-			&nodes.EdgeDoUseIdentity{Identity: node.Identity},
+			&nodes.EdgeDoVerifyIdentity{
```

Comment:
There are other places using `EdgeDoVerifyIdentity`: `add_identity`, `update_identity`, `verify_identity`, `verify_identity_resume`. Can you help to fix them too? Thanks!